src/components/Badges/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+
+import * as S from "./styles"
+import { Props } from "./types"
+
+import { Box } from "../Box"
+
+export const Badges = ({
+    label,
+    type = "badge",
+    startIcon,
+    endIcon
+}: Props) => {
+    const [showCloseIcon, setShowCloseIcon] = useState(false)
+    const [closeTag, setCloseTag] = useState(false)
+
+    return (
+        <Box>
+            {type === "tag" ? (
+                <S.Tags
+                    onMouseEnter={() => setShowCloseIcon(!showCloseIcon)}
+                    onMouseLeave={() => setShowCloseIcon(!showCloseIcon)}
+                    style={closeTag ? { display: "none" } : {}}
```

Comment:
1. Usar prop `css`
2. Usar seguinte sintaxe para atribuir propriedade de forma condicional de modo mais conciso: `...(closeTag && {b: 5})`

```suggestion
                    css={...(closeTag && {display: "none"})}
```